src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+html {
+  height: 100%;
+}
+
 body {
   margin: 0;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
The body is set to `display: flex;` with `justify-content: center;` and `align-items: center;`. This is a great way to center content both vertically and horizontally. However, it's important to note that this will center all direct children of the body. If you don't want all elements to be centered, you might need to wrap the content you want to center in a separate div.